components/core/src/GlobalSQLiteMetadataDB.hpp

```diff
@@ -71,7 +71,7 @@ class GlobalSQLiteMetadataDB : public GlobalMetadataDB {
                               uint64_t size) override;
     void update_metadata_for_files (const std::string& archive_id, const std::vector<streaming_archive::writer::File*>& files) override;
 
-    GlobalMetadataDB::ArchiveIterator* get_archive_iterator () override { return new ArchiveIterator(m_db); }
+    GlobalMetadataDB::ArchiveIterator* get_archive_iterator (epochtime_t begin_ts, epochtime_t end_ts) override { return new ArchiveIterator(m_db); }
```

Comment:
Do you see any concerns with implementing the time range filtering for SQLite databases?

Reply:
No concerns really, I've added the implementation for SQLite database as well.